s3auth-hosts/src/test/java/com/s3auth/hosts/DefaultResourceTest.java

```diff
@@ -367,4 +368,37 @@ public void getsContentEncodingHeaderFromAmazonObject() throws Exception {
         );
     }
 
+    /**
+     * Reproduces the scenario when an Amazon S3 bucket connection throws
```

Comment:
please see http://www.yegor256.com/2014/04/27/typical-mistakes-in-java-code.html#test-method-names